backend/fms_core/services/study.py

```diff
@@ -80,3 +99,19 @@ def get_study(project_obj: Project, study_letter: str):
         errors.append(f"Both a project and a study letter are required to retrieve a study.")
 
     return study, errors, warnings
+
+def can_remove_study(study: int) -> Tuple[bool, Dict[str, List[str]], Dict[str, List[str]]]:
```

Comment:
If you pass an int here you are not passing a Study but an int. Rename param to study_id to clarify